src/EditorFeatures/CSharpTest/Diagnostics/FixAllProvider/BatchFixerTests.cs

```diff
@@ -43,7 +43,7 @@ private static void AnalyzeNode(SyntaxNodeAnalysisContext context)
                     var symbol = context.SemanticModel.GetSymbolInfo(node).Symbol;
                     if (symbol != null && symbol.Kind == SymbolKind.Field)
                     {
-                        var diagnostic = Diagnostic.Create(Descriptor, node.GetLocation());
+                        var diagnostic = CodeAnalysis.Diagnostic.Create(Descriptor, node.GetLocation());
```

Comment:
Odd.  was this necessary?

Reply:
Yes. I had to make `AbstractCodeActionOrUserDiagnosticTest` from`TestBase` to access test extensions for `Diagnostic` and `DiagnosticDescription` for diagnostic-only tests for IDE0052. There is a protected static method called `Diagnostic` in `TestBase`, which caused conflicts.

---
In reply to: [213405994](https://github.com/dotnet/roslyn/pull/29511#discussion_r213405994) [](ancestors = 213405994)